### DISCUSSION TITLE
COMBOOT without Legacy BIOS

### DISCUSSION THREAD

**Discussion by shredstick** [ipxe/ipxe#496] [Q&A] [answered]
Hi Everyone.

I have read that COMBOOT will not work with UEFI ipxe.
And the problem i have is that our new Notebooks only support UEFI, not Legacy BIOS.
I saw that there is also a way to create a Linux Binary which im not familiar with. Could that be a way to overcome this problem?
If theres no other option I think I would have to change from COMBOOT to something else and I don't know if my Distribution Server even supports something else.

Any help is welcome. Thanks a lot.

**Comment by NiKiZe** [comment, marked as answer]
Comboot uses pcbios calls, something that does not exist in EFI.
https://wiki.syslinux.org/wiki/index.php?title=Doc/comboot#COMBOOT_file_format
